pkg/gofr/logging/remotelogger/dynamic_level_logger_test.go

```diff
@@ -32,7 +32,7 @@ func TestRemoteLogger_UpdateLevel(t *testing.T) {
 
 	go rl.UpdateLogLevel()
 
-	time.Sleep(2 * time.Second)
+	time.Sleep(1200 * time.Millisecond)
```

Comment:
I think this one can be reduced with the changes you made

Reply:
Sure reduced